src/rust/engine/BUILD

```diff
@@ -0,0 +1,17 @@
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# Track files depended on to build the native engine to allow for downstream invalidations.
+files(
+  sources=zglobs(
+    '**/Cargo.*',
+    '**/*.rs',
+    exclude=[
+      zglobs('**/target/*'),
+      'process_execution/bazel_protos'
+    ]
+  ),
+  dependencies=[
+    'src/rust/engine/process_execution/bazel_protos'
+  ]
+)
```

Comment:
Were these discovered while attempting to enable `--chroot`? Would be super awesome to get there...

Reply:
No, but I had realized these should be added when shepherding @cosmicexplorer's change that introduced the idea of adding implicit deps to `int-test`. Doing cleanups yesterday I popped this TODO off my stack.